src/shaders/glsl/mesh.fragment.shaded.js

```diff
@@ -16,7 +18,12 @@ vec4 getShadedColor(vec4 rgba) {
   vec3 light = normalize(vLight);
   vec3 position = normalize(vPosition);
   
-  float side    = gl_FrontFacing ? -1.0 : 1.0;
```

Comment:
@sritchie If you feel like futzing with this, try using `gl_FrontFacing ? +1.0 : - 1.0`...you'll see both sides of the surface turn light.

Reply:
This one seems to bite many folks. I've been reading up on it, and would NOT have found this on my own. Great find on this fix.